docs/src/main/asciidoc/databases-dev-services.adoc

```diff
@@ -0,0 +1,90 @@
+Don't configure a database URL, username and password - Quarkus will provide the database and you can just start coding without worrying about config.
+
+Production databases need to be configured as normal, so if you want to include a production database config in your
+`application.properties` and continue to use Dev Services we recommend that you use the `%prod.` profile to define your database settings.
+
+== Enabling / Disabling Dev Services for Database
+
+Dev Services for databases automatically starts a database server in dev mode and when running tests.
+So, you don't have to start a server manually.
+The application is configured automatically.
+
+You can disable the automatic database start in `application.properties` via:
+
+[source,properties]
+----
+quarkus.devservices.enabled=false
+# OR
+quarkus.datasource.devservices.enabled=false
+----
+
+Dev Services for databases relies on Docker to start the server (except for H2 and Derby which are run in process).
+If your environment does not support Docker, you will need to start the server manually, or connect to an already running server.
+
+[[license_acceptance]]
+=== Proprietary Databases - License Acceptance
+
+If you are using a proprietary database such as DB2 or MSSQL you will need to accept the license agreement.
+To do this create a `src/main/resources/container-license-acceptance.txt` files in your project and add a line with the image name and tag of the database.
+By default, Quarkus uses the default image for the current version of Testcontainers, if you attempt to start Quarkus the resulting failure will tell you the exact image name in use for you to add to the file.
+
+An example file is shown below:
+
+.src/main/resources/container-license-acceptance.txt
+----
+ibmcom/db2:11.5.0.0a
+mcr.microsoft.com/mssql/server:2017-CU12
+----
+
+== Database Vendor Specific Configuration
+
+All services based on containers are run using Testcontainers but Quarkus is not using the Testcontainers JDBC driver.
+Thus, even though extra JDBC URL properties can be set in your `application.properties` file, specific properties supported by the Testcontainers JDBC driver such as `TC_INITSCRIPT`, `TC_INITFUNCTION`, `TC_DAEMON`, `TC_TMPFS` are not supported.
+
+Quarkus can support *specific* properties sent to the container itself though, e.g. this is the case for `TC_MY_CNF` which allows to override the MariaDB/MySQL configuration file.
+
+Overriding the MariaDB/MySQL configuration would be done as follows:
+
+[source,properties]
+----
+quarkus.datasource.devservices.container-properties.TC_MY_CNF=testcontainers/mysql-conf
+----
+
+PostgreSQL docker images can be configured via docker command (see https://github.com/docker-library/docs/blob/master/postgres/README.md#database-configuration[the official docu]).
+For the PostgreSQL Dev Service the command can be set like this:
```

Comment:
```suggestion
For instance, a command could be set like this:
```

Reply:
I took out the whole part. Will be added back during https://github.com/quarkusio/quarkus/pull/25289